dist/js/classes/player.js

```diff
@@ -615,15 +615,15 @@ class Player {
         return Math.max(crit, 0);
     }
     getDodgeChance(weapon) {
-        return Math.max((5 + (this.target.defense - this.stats.skill) * 0.1) - this.stats.exp * this.EXP_RATING_COEFFICIENT, 0);
+        return Math.max((5 + (this.target.defense - this.stats.skill) * 0.1) - Math.floor(this.stats.exp * this.EXP_RATING_COEFFICIENT), 0);
     }
 
     getParryChance(weapon) {
         if (!this.activetank) {
             return 0;
         }
         else {
-            return Math.max(14 - this.stats.exp * this.EXP_RATING_COEFFICIENT, 0);
+            return Math.max(14 - Math.floor(this.stats.exp * this.EXP_RATING_COEFFICIENT), 0);
```

Comment:
Maybe I'm misreading it, but this expression results in the same result for expertise ratings 20, 25, and 30, for example (14 - 1 in all cases). I don't think this is the intended behavior -- is the intention to introduce expertise jumps by 5?

Reply:
The current ``EXP_RATING_COEFFICIENT`` was falsely attributed with 1/10*82/52 or the same as ``HIT_RATING_COEFFICIENT``. I fixed that along with how ``exp`` is converted to dodge reduction on 4a933f7.

Using no expertise except Grip of Deftness (18 expertise rating) as an example:
<details>

18 expertise rating results in ~1.13% dodge / parry reduction in master branch.
<summary>In master branch</summary>

![image](https://user-images.githubusercontent.com/47816228/132785360-367f3872-11d8-48c5-bf8d-461437036df6.png)

</details>

18 expertise rating results in exactly 1% dodge / parry reduction in this PR's branch.
<details>

<summary>In this PR's branch</summary>

![image](https://user-images.githubusercontent.com/47816228/132785411-47e0cbe8-4fac-4079-b9bf-22ec3e4d750f.png)

</details>